keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -234,21 +234,81 @@ def test_saved_model(self, save_format, filename):
 
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
-    def test_backbone_preset_weight_loading(self):
-        # Check that backbone preset weights loaded correctly
-        # TODO(lukewood): need to forward pass test once proper weights are
-        # implemented
-        keras_cv.models.RetinaNet.from_preset(
-            "resnet50_v2_imagenet",
-            num_classes=20,
-            bounding_box_format="xywh",
-        )
+    def test_backbone_preset(self):
+        weights = [
+            "csp_darknet_tiny",
```

Comment:
I don't think we need to load every preset for every backbone, just 1 preset per backbone should be good (preferably the smallest preset for every backbone)